install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml

```diff
@@ -136,6 +136,14 @@ data:
               fieldPath: metadata.name
         - name: ISTIO_META_INTERCEPTION_MODE
           value: {{ "[[ or (index .ObjectMeta.Annotations \"sidecar.istio.io/interceptionMode\") .ProxyConfig.InterceptionMode.String ]]" }}
+        {{ "[[ range $k,$v := .ObjectMeta.Annotations ]]" }}
+        - name: ISTIO_META_ANN_{{ "[[ $k ]]" }}
+          value: {{ "[[ $v ]]" }}
```

Comment:
The value should be serialized / b64 encoded. annotations can be structs and simply pasting them here will fail.